class.jetpack-cli.php

```diff
@@ -559,6 +559,184 @@ public function options( $args, $assoc_args ) {
 				break;
 		}
 	}
+
+	public function sync( $args, $assoc_args ) {
```

Comment:
Let's add some documentation to this command, so that `wp jetpack sync --help` will be helpful :) 

If you follow the syntax of one of the other commands, you should get a nice helpful message back